src/interpolation/compute/compute.test.js

```diff
@@ -81,24 +99,39 @@ test('Unknown formatter', () => {
   expect(interpolated).toBe('Thibaud')
 })
 
-test('Unknown formatter in middle is ignored', () => {
+test('Unknown formatter in middle', () => {
   const interpolated = compute({ name: 'Thibaud' }, 'name | smurf | lowercase')
 
   expect(interpolated).toBe('thibaud')
 })
 
-test('Option formatters', () => {
-  const interpolated = compute({ name: 'Thibaud' }, 'name | smurf', {
-    formatters: { smurf: txt => 'Smurf!' }
-  })
+// scrict: true
+test('Unknown variable // Strict: true', () => {
```

Comment:
Since you don't write "Strict:true", you can only put: `Unknown variable - strict`